azext_iot/central/_help.py

```diff
@@ -212,6 +212,45 @@ def _load_central_users_help():
             --object-id {objectId}
             --role operator
     """
+    helps[
+        "iot central app user show"
+    ] = """
+    type: command
+    short-summary: Get the details of a user by ID
+    examples:
+      - name: Get details of user
+        text: >
+          az iot central app user show
+          --app-id {appid}
+          --id {userId}
+    """
+
+    helps[
+        "iot central app user delete"
+    ] = """
+    type: command
+    short-summary: Delete a user from the application
+    examples:
+      - name: Delete a user
+        text: >
+          az iot central app user delete
+          --app-id {appid}
+          --id {userId}
```

Comment:
I think it would be helpful to add `--user-id` as an alias for `--id` in this command group as `--id` is so generic.